rust/cubesql/cubeclient/src/apis/default_api.rs

```diff
@@ -140,9 +140,16 @@ pub async fn meta_v1(
         serde_json::from_str(&local_var_content).map_err(Error::from)
     } else {
         let local_var_entity: Option<MetaV1Error> = serde_json::from_str(&local_var_content).ok();
+        let sanitized_error_content: String = match local_var_entity.clone() {
```

Comment:
@ovr Not entirely sure I should be doing this here, as this file can be re-generated. Is there a better place to do this?

Reply:
It's better to formate this on transforming, https://github.com/cube-js/cube.js/blob/master/rust/cubesql/cubesql/src/lib.rs#L89

Or on call.